mirror_pso_containers.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Script to mirror PSO required containers into a local Registry
+
+# Ensure you have logged in to your local registry using 'docker login' before using this script
+
+REGISTRY_URL=<your local registry address, including port>
```

Comment:
[Trivial]
It would be great if there's one extra line of comment to show an example.